features/internal/hvac.go

```diff
@@ -0,0 +1,51 @@
+package internal
+
+import (
+	spineapi "github.com/enbility/spine-go/api"
+	"github.com/enbility/spine-go/model"
+)
+
+type HvacCommon struct {
+	featureLocal  spineapi.FeatureLocalInterface
+	featureRemote spineapi.FeatureRemoteInterface
+}
+
+// NewLocalHvac creates a new HvacCommon helper for local entities
+func NewLocalHvac(featureLocal spineapi.FeatureLocalInterface) *HvacCommon {
+	return &HvacCommon{
+		featureLocal: featureLocal,
+	}
+}
+
+// NewRemoteHvac creates a new HvacCommon helper for remote entities
+func NewRemoteHvac(featureRemote spineapi.FeatureRemoteInterface) *HvacCommon {
+	return &HvacCommon{
+		featureRemote: featureRemote,
+	}
+}
+
+// GetHvacOperationModeDescriptions returns the operation mode descriptions
+func (h *HvacCommon) GetHvacOperationModeDescriptions() ([]model.HvacOperationModeDescriptionDataType, error) {
+	function := model.FunctionTypeHvacOperationModeDescriptionListData
+	operationModeDescriptions := make([]model.HvacOperationModeDescriptionDataType, 0)
+
+	data, err := featureDataCopyOfType[model.HvacOperationModeDescriptionListDataType](h.featureLocal, h.featureRemote, function)
+	if err == nil || data != nil {
+		operationModeDescriptions = append(operationModeDescriptions, data.HvacOperationModeDescriptionData...)
+	}
+
+	return operationModeDescriptions, nil
+}
+
+// GetHvacSystemFunctionOperationModeRelations returns the operation mode relations (used to map operation modes to setpoints)
+func (h *HvacCommon) GetHvacSystemFunctionOperationModeRelations() ([]model.HvacSystemFunctionSetpointRelationDataType, error) {
```

Comment:
Fix the function name. It needs to be `GetHvacSystemFunctionSetpointRelations` instead of `GetHvacSystemFunctionOperationModeRelations `